tests/test_runner.py

```diff
@@ -485,7 +488,9 @@ def test_await_syntax_error():
                         {
                             "type": "syntax_error",
                             "text": (
-                                '  File "my_program.py", line 1\n'
+                                f'  File "{filename}", line 1\n'
+                                '    await b\n'
+                                '    ^\n'
```

Comment:
I don't know why this wasn't there before, seems like you might have fixed a bug somehow.

Reply:
Probably because of being backed by a file. I'm not entirely source how the errors get resolved, but I think linecache wasn't enough by itself, which is why I started using it in Papyros.